pkg/ctl/delete/nodegroup.go

```diff
@@ -136,7 +142,10 @@ func doDeleteNodeGroup(cmd *cmdutils.Cmd, ng *api.NodeGroup, updateAuthConfigMap
 		if !cmd.Plan {
 			for _, ng := range allNodeGroups {
 				if err := drain.NodeGroup(clientSet, ng, ctl.Provider.WaitTimeout(), maxGracePeriod, false); err != nil {
-					return err
+					if !force {
+						return err
+					}
```

Comment:
I think we should only ignore drain errors if the nodegroup's CloudFormation stack is not in a normal state. It's still possible for `drain.NodeGroup` to return an error if the nodegroup is healthy, and we should propagate that error and stop the deletion process.

Reply:
Makes sense! I'll change the logic

Reply:
Added a hint instead to suggest they can use the `--drain=false` flag to skip the error